gerrit/changeset.go

```diff
@@ -32,24 +32,22 @@ func MakeChangeset(changeInfo *goGerrit.ChangeInfo) *Changeset {
 		Number:          changeInfo.Number,
 		Verified:        labelInfoToInt(changeInfo.Labels["Verified"]),
 		CodeReviewed:    labelInfoToInt(changeInfo.Labels["Code-Review"]),
+		Autosubmit:      labelInfoToInt(changeInfo.Labels["Autosubmit"]),
```

Comment:
I'm not confident about how this behaves on systems without this Label created. Maybe we can add a test, passing in a `goGerrit.ChangeInfo` without the label set and ensure `IsAutosubmit()` returns false?

Maybe we should do some pre-flight check to startup to test this (if we're able to get this information from Gerrit).